app/models/AddressJourneyTTLModels.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json._
+import reactivemongo.bson.BSONDateTime
+import reactivemongo.play.json._
+
+case class AddressJourneyTTLModel(nino: String, editedAddress: EditedAddress)
+
+sealed trait EditedAddress {
+  val expireAt: BSONDateTime
+  def getIndex: String
```

Comment:
get index doesn't make to much sense as to what the value is. Maybe something like `addressType` would be better